conjur-oss/values.yaml

```diff
@@ -7,6 +7,7 @@
 # rather than setting these in a custom values YAML file. This avoids the
 # risk of leaving around residual values files containing this sensitive
 # information.
+dataKeySecretRef: "conjur-data-key"
```

Comment:
I think this should default to an empty string. That way if it's not explicitly provided it will fail validation. With this default, I think it would succeed initially and then fail when it couldn't find this secret, which would lead to a confusing error message.

Reply:
Currently, it fails because two of the values are provided, so I'll just remove it altogether